aeron-client/src/main/java/io/aeron/AeronCounters.java

```diff
@@ -279,6 +279,43 @@
      */
     public static final int CLUSTER_STANDBY_CONTROL_TOGGLE_TYPE_ID = 223;
 
+    /**
+     * The type id of the {@link Counter} used for keeping track of the max duty cycle time of the cluster standby.
+     */
+    public static final int CLUSTER_STANDBY_MAX_CYCLE_TIME_TYPE_ID = 227;
+
+    /**
+     * The type id of the {@link Counter} used for keeping track of the count of cycle time threshold exceeded of
+     * the cluster standby.
+     */
+    public static final int CLUSTER_STANDBY_CYCLE_TIME_THRESHOLD_EXCEEDED_TYPE_ID = 228;
+
+    /**
+     * Counter type id for the transition module error count.
+     */
+    public static final int TRANSITION_MODULE_ERROR_COUNT_TYPE_ID = 226;
+
+    /**
+     * The type if of the {@link Counter} used for transition module state
+     */
+    public static final int TRANSITION_MODULE_STATE_TYPE_ID = 224;
```

Comment:
Why are ids are not sorted? If one scans from top to bottom it looks like some are missing or have been deleted.

Reply:
I've grouped them by name rather than in counter id order.  This seemed tidier to me.  I've added a test to ensure that we don't have accidentally add clashing ids.